com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```diff
@@ -442,7 +434,7 @@ modules:
     subdir: vkd3d-proton
     buildsystem: meson
     build-options:
-      ldflags: &mingw-ldflags ""
```

Comment:
I wonder why did it ate this non-empty anchor. Not critical, but this feels wrong.

Reply:
@wjt Newer `ruamel.yaml` doesn't seem to eat anchors. Do you think it worth updating it in the docker image?

Reply:
If it fixes a bug, sounds good!

Reply:
Well, yaml anchors aren't widely used on Flathub. Mostly used by me. And we'll have to switch from Debian package to PyPI package for that case.